xform-marc21-to-xml/src/main/java/edu/stanford/MarcXMLValidator.java

```diff
@@ -0,0 +1,97 @@
+package edu.stanford;
+
+import org.apache.commons.cli.*;
+import org.xml.sax.SAXException;
+
+import javax.xml.XMLConstants;
+import javax.xml.transform.Source;
+import javax.xml.transform.stream.StreamSource;
+import javax.xml.validation.Schema;
+import javax.xml.validation.SchemaFactory;
+import javax.xml.validation.Validator;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+
+/**
+ *
+ */
+public class MarcXMLValidator {
+
+    // Apache Commons-CLI Options
+    // https://commons.apache.org/proper/commons-cli/introduction.html
+    static CommandLine cmd = null;
+    static Options options = setOptions();
+
+    static Options setOptions() {
+        Options opts = new Options();
+        opts.addOption("h", "help", false, "help message");
+        opts.addOption("i", "inputFile", true, "MARC-XML input file");
+        return opts;
+    }
+
+    static void printHelp() {
+        HelpFormatter formatter = new HelpFormatter();
+        formatter.printHelp(MarcXMLValidator.class.getName(), options);
+    }
+
+    static String parseInputFile(String iFile) {
```

Comment:
There's no reasonable class hierarchy between those classes, so they can't be subclasses of a common superclass that provides this parse method for an input file.  The MarcToXML already has a superclass and java doesn't allow multiple inheritance.  An interface would not work, because it only has abstract methods, so each class has to implement it anyway.  So, good suggestion but I just don't see how it could work that way.  It's possible that both classes could use another class that only does some form of command line option setting and parsing, but that seems like a bit of indirection that doesn't gain a lot.  Maybe the only option that might work is a new class with one static method on it that is this method?